### ISSUE TITLE
undefined method error occurred when "remove tag" is executed with no arguments

### ISSUE THREAD

**Issue by m-t-a-n-a-k-a** [JacobEvelyn/friends#262]
VERSION
0.51（ruby 2.6.0）

SUMMARY
When "remove tag" is executed with no arguments,  the following error messages are output.
```
$friends remove tag
Error: undefined method `strip' for nil:NilClass
```

When "remove nickname”, the error messages are as below.
```
$friends remove nickname
Error: No friend found for ""
```

I think it would be better to output "Error: No friend found for" message in the case of "remove tag" as well. 👍

**Comment by JacobEvelyn** [comment]
Great catch! Thank you for finding this (and sorry for the slow response)! ❤️

**Comment by JacobEvelyn** [comment]
Thanks so much for fixing this, @m-t-a-n-a-k-a! Your fix is now released in version 0.52! 🎉
